include/boost/graph/hawick_circuits.hpp

```diff
@@ -366,21 +385,24 @@ void hawick_circuits(BOOST_FWD_REF(Graph) graph, BOOST_FWD_REF(Visitor) visitor)
 template < typename Graph, typename Visitor, typename VertexIndexMap >
 void hawick_unique_circuits(BOOST_FWD_REF(Graph) graph,
     BOOST_FWD_REF(Visitor) visitor,
-    BOOST_FWD_REF(VertexIndexMap) vertex_index_map)
+    BOOST_FWD_REF(VertexIndexMap) vertex_index_map,
+    unsigned int const maxLength = 0)
```

Comment:
```suggestion
    unsigned int maxLength = 0)
```

Reply:
Thanks, I am removing the `const` from the `max_length` function arguments, plus some minor cleanup (like `max_length` instead of `maxLength`).

One question: what about the data member `max_length_`? Should that remain `const`?